buildout.cfg

```diff
@@ -27,4 +27,6 @@ eggs = ZServer
 
 [scripts]
 recipe = zc.recipe.egg
-eggs = ZServer
+eggs =
+    Zope
```

Comment:
Zope is an explicit dependency of ZServer:  we shouldn't need to call it out here.

Reply:
Whoops, accidentally left that in from local testing, did not mean to commit buildout.cfg.